pkg/index/types.go

```diff
@@ -62,4 +62,17 @@ type FileOperation struct {
 // Receipt describes a plugin receipt file.
 type Receipt struct {
 	Plugin `json:",inline" yaml:",inline"`
+
+	Status ReceiptStatus `json:"status,omitempty"`
```

Comment:
why `,omitempty`? we want receipts to always have `status` (except today's receipts, which currently don't have, but once they are updated, they should).

Reply:
This was just me misremembering what we talked about before, I wrote up the receipt issue before you and I cleared that up and I copied over the struct definitions I wrote there and forgot to remove this. I'll get rid of it :+1: